### DISCUSSION TITLE
如何将封面“副导师”与“导师”对齐？

### DISCUSSION THREAD

**Discussion by Nemours-Winfty** [sjtug/SJTUThesis#1042] [提问]
<img width="151" alt="截屏2024-05-24 10 36 06" src="https://github.com/sjtug/SJTUThesis/assets/110967756/ddcf1ed1-a263-46c9-aff2-20e3a275f03d">
直接套模版的生成结果如图所示，请问怎样能让“副导师”与“导师”对齐？

**Comment by NemoYuan2008** [comment]
你好！请问你使用的是来自 GitHub 的模板吗？该模板直接编译的话是对齐的：
![image](https://github.com/sjtug/SJTUThesis/assets/34743600/835ecb92-f2c7-4896-8119-fafedb2291ab)

**Reply by Nemours-Winfty** [reply]
你好，我是直接用的bysj.jwc.sjtu.edu.cn给的LaTeX模版，出来就是这样了
<img width="1079" alt="截屏2024-05-24 17 57 18" src="https://github.com/sjtug/SJTUThesis/assets/110967756/ff6efdc2-60bf-4030-bba2-890b0b0af11b">
（图里个人信息涂白了）
